packages/effection/tests/order-of-operations.test.js

```diff
@@ -24,17 +24,15 @@ describe('Order of Shutdown Operations', () => {
   beforeEach(() => {
     order = [];
 
-    let self = ({ resume, context: { parent }}) => resume(parent);
```

Comment:
The reason I had to rewrite this is because returning the `parent` here actually transfers ownership of it now 😱